src/rust/Cargo.toml

```diff
@@ -34,7 +34,7 @@ members = [
 ]
 
 [profile.dev]
-opt-level = 0
+opt-level = 2
```

Comment:
@wdanilo this can be reverted. But I am wondering: If we are internally using this to speed up development iterations, why don't we push it to `main`?